homeassistant/components/solarlog/strings.json

```
@@ -11,10 +11,10 @@
     },
     "error": {
       "already_configured": "Device is already configured",
```

```suggestion
      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
```
Can the abort keys be used here?

---

Yes it can be used here.

---

Thanks for the reply. 👍